python: can't open file 'apy.py': [Errno 2] No such file or directory
 * Running on http://0.0.0.0:5000/
 * Restarting with reloader
121.54.32.171 - - [08/Feb/2015 08:55:28] "POST /imageSearch HTTP/1.1" 200 -
121.54.32.171 - - [08/Feb/2015 10:57:29] "POST /imageSearch HTTP/1.1" 200 -
 * Detected change in 'api.py', reloading
 * Restarting with reloader
121.54.32.171 - - [08/Feb/2015 11:46:18] "POST /imageSearch HTTP/1.1" 200 -
 * Running on http://0.0.0.0:5000/
 * Restarting with reloader
 * Detected change in 'api.py', reloading
 * Restarting with reloader
121.54.32.163 - - [08/Feb/2015 13:19:13] "POST /imageSearch HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python2.7/dist-packages/flask_cors.py", line 272, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask_cors.py", line 272, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/var/www/html/basic/api/api.py", line 17, in imageSearch
    result = diseases(filename)
  File "/var/www/html/basic/api/api.py", line 93, in diseases
    return json.dumps(val)
  File "/usr/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: array([[   7.],
       [   7.],
       [   9.],
       [  34.],
       [  53.],
       [  84.],
       [ 116.],
       [ 131.],
       [ 114.],
       [ 123.],
       [ 166.],
       [ 168.],
       [ 117.],
       [  82.],
       [  85.],
       [  60.],
       [  57.],
       [  71.],
       [  59.],
       [  59.],
       [  48.],
       [  62.],
       [  81.],
       [ 104.],
       [  82.],
       [ 102.],
       [  91.],
       [ 110.],
       [ 127.],
       [ 114.],
       [ 140.],
       [ 149.],
       [ 149.],
       [ 192.],
       [ 208.],
       [ 238.],
       [ 270.],
       [ 270.],
       [ 306.],
       [ 313.],
       [ 372.],
       [ 384.],
       [ 411.],
       [ 414.],
       [ 461.],
       [ 437.],
       [ 479.],
       [ 467.],
       [ 473.],
       [ 510.],
       [ 527.],
       [ 576.],
       [ 606.],
       [ 612.],
       [ 595.],
       [ 647.],
       [ 691.],
       [ 674.],
       [ 641.],
       [ 705.],
       [ 733.],
       [ 738.],
       [ 748.],
       [ 733.],
       [ 726.],
       [ 757.],
       [ 780.],
       [ 805.],
       [ 795.],
       [ 823.],
       [ 829.],
       [ 766.],
       [ 775.],
       [ 844.],
       [ 870.],
       [ 861.],
       [ 822.],
       [ 832.],
       [ 885.],
       [ 805.],
       [ 867.],
       [ 844.],
       [ 814.],
       [ 841.],
       [ 835.],
       [ 800.],
       [ 865.],
       [ 884.],
       [ 830.],
       [ 903.],
       [ 894.],
       [ 812.],
       [ 864.],
       [ 803.],
       [ 866.],
       [ 837.],
       [ 821.],
       [ 810.],
       [ 822.],
       [ 788.],
       [ 767.],
       [ 823.],
       [ 804.],
       [ 759.],
       [ 766.],
       [ 785.],
       [ 773.],
       [ 744.],
       [ 744.],
       [ 749.],
       [ 706.],
       [ 671.],
       [ 685.],
       [ 721.],
       [ 739.],
       [ 658.],
       [ 627.],
       [ 630.],
       [ 623.],
       [ 604.],
       [ 539.],
       [ 586.],
       [ 532.],
       [ 507.],
       [ 467.],
       [ 472.],
       [ 514.],
       [ 470.],
       [ 487.],
       [ 446.],
       [ 387.],
       [ 413.],
       [ 370.],
       [ 358.],
       [ 336.],
       [ 339.],
       [ 327.],
       [ 288.],
       [ 266.],
       [ 272.],
       [ 248.],
       [ 228.],
       [ 223.],
       [ 199.],
       [ 227.],
       [ 190.],
       [ 161.],
       [ 197.],
       [ 155.],
       [ 147.],
       [ 135.],
       [ 123.],
       [ 124.],
       [ 107.],
       [  93.],
       [  99.],
       [  66.],
       [  86.],
       [  72.],
       [  55.],
       [  67.],
       [  66.],
       [  60.],
       [  53.],
       [  58.],
       [  43.],
       [  36.],
       [  40.],
       [  39.],
       [  31.],
       [  33.],
       [  45.],
       [  36.],
       [  28.],
       [  32.],
       [  31.],
       [  26.],
       [  27.],
       [  27.],
       [  28.],
       [  32.],
       [  28.],
       [  32.],
       [  36.],
       [  20.],
       [  29.],
       [  34.],
       [  44.],
       [  24.],
       [  32.],
       [  41.],
       [  37.],
       [  41.],
       [  29.],
       [  36.],
       [  46.],
       [  32.],
       [  35.],
       [  44.],
       [  43.],
       [  43.],
       [  48.],
       [  52.],
       [  61.],
       [  66.],
       [  62.],
       [  61.],
       [  51.],
       [  53.],
       [  53.],
       [  49.],
       [  38.],
       [  40.],
       [  31.],
       [  25.],
       [  32.],
       [  42.],
       [  35.],
       [  24.],
       [  42.],
       [  36.],
       [  38.],
       [  29.],
       [  36.],
       [  32.],
       [  32.],
       [  26.],
       [  33.],
       [  16.],
       [  18.],
       [  20.],
       [  17.],
       [  13.],
       [  11.],
       [  15.],
       [   9.],
       [  16.],
       [  15.],
       [  11.],
       [   7.],
       [   9.],
       [   4.],
       [   6.],
       [   7.],
       [  10.],
       [   3.],
       [  10.],
       [   3.],
       [   8.],
       [   7.],
       [   9.],
       [   4.],
       [   8.],
       [   5.],
       [   8.],
       [   4.]], dtype=float32) is not JSON serializable
121.54.32.163 - - [08/Feb/2015 13:20:30] "POST /imageSearch HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python2.7/dist-packages/flask_cors.py", line 272, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask_cors.py", line 272, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/var/www/html/basic/api/api.py", line 17, in imageSearch
    result = diseases(filename)
  File "/var/www/html/basic/api/api.py", line 93, in diseases
    return json.dumps(val)
  File "/usr/lib/python2.7/json/__init__.py", line 243, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: array([[   7.],
       [   7.],
       [   9.],
       [  34.],
       [  53.],
       [  84.],
       [ 116.],
       [ 131.],
       [ 114.],
       [ 123.],
       [ 166.],
       [ 168.],
       [ 117.],
       [  82.],
       [  85.],
       [  60.],
       [  57.],
       [  71.],
       [  59.],
       [  59.],
       [  48.],
       [  62.],
       [  81.],
       [ 104.],
       [  82.],
       [ 102.],
       [  91.],
       [ 110.],
       [ 127.],
       [ 114.],
       [ 140.],
       [ 149.],
       [ 149.],
       [ 192.],
       [ 208.],
       [ 238.],
       [ 270.],
       [ 270.],
       [ 306.],
       [ 313.],
       [ 372.],
       [ 384.],
       [ 411.],
       [ 414.],
       [ 461.],
       [ 437.],
       [ 479.],
       [ 467.],
       [ 473.],
       [ 510.],
       [ 527.],
       [ 576.],
       [ 606.],
       [ 612.],
       [ 595.],
       [ 647.],
       [ 691.],
       [ 674.],
       [ 641.],
       [ 705.],
       [ 733.],
       [ 738.],
       [ 748.],
       [ 733.],
       [ 726.],
       [ 757.],
       [ 780.],
       [ 805.],
       [ 795.],
       [ 823.],
       [ 829.],
       [ 766.],
       [ 775.],
       [ 844.],
       [ 870.],
       [ 861.],
       [ 822.],
       [ 832.],
       [ 885.],
       [ 805.],
       [ 867.],
       [ 844.],
       [ 814.],
       [ 841.],
       [ 835.],
       [ 800.],
       [ 865.],
       [ 884.],
       [ 830.],
       [ 903.],
       [ 894.],
       [ 812.],
       [ 864.],
       [ 803.],
       [ 866.],
       [ 837.],
       [ 821.],
       [ 810.],
       [ 822.],
       [ 788.],
       [ 767.],
       [ 823.],
       [ 804.],
       [ 759.],
       [ 766.],
       [ 785.],
       [ 773.],
       [ 744.],
       [ 744.],
       [ 749.],
       [ 706.],
       [ 671.],
       [ 685.],
       [ 721.],
       [ 739.],
       [ 658.],
       [ 627.],
       [ 630.],
       [ 623.],
       [ 604.],
       [ 539.],
       [ 586.],
       [ 532.],
       [ 507.],
       [ 467.],
       [ 472.],
       [ 514.],
       [ 470.],
       [ 487.],
       [ 446.],
       [ 387.],
       [ 413.],
       [ 370.],
       [ 358.],
       [ 336.],
       [ 339.],
       [ 327.],
       [ 288.],
       [ 266.],
       [ 272.],
       [ 248.],
       [ 228.],
       [ 223.],
       [ 199.],
       [ 227.],
       [ 190.],
       [ 161.],
       [ 197.],
       [ 155.],
       [ 147.],
       [ 135.],
       [ 123.],
       [ 124.],
       [ 107.],
       [  93.],
       [  99.],
       [  66.],
       [  86.],
       [  72.],
       [  55.],
       [  67.],
       [  66.],
       [  60.],
       [  53.],
       [  58.],
       [  43.],
       [  36.],
       [  40.],
       [  39.],
       [  31.],
       [  33.],
       [  45.],
       [  36.],
       [  28.],
       [  32.],
       [  31.],
       [  26.],
       [  27.],
       [  27.],
       [  28.],
       [  32.],
       [  28.],
       [  32.],
       [  36.],
       [  20.],
       [  29.],
       [  34.],
       [  44.],
       [  24.],
       [  32.],
       [  41.],
       [  37.],
       [  41.],
       [  29.],
       [  36.],
       [  46.],
       [  32.],
       [  35.],
       [  44.],
       [  43.],
       [  43.],
       [  48.],
       [  52.],
       [  61.],
       [  66.],
       [  62.],
       [  61.],
       [  51.],
       [  53.],
       [  53.],
       [  49.],
       [  38.],
       [  40.],
       [  31.],
       [  25.],
       [  32.],
       [  42.],
       [  35.],
       [  24.],
       [  42.],
       [  36.],
       [  38.],
       [  29.],
       [  36.],
       [  32.],
       [  32.],
       [  26.],
       [  33.],
       [  16.],
       [  18.],
       [  20.],
       [  17.],
       [  13.],
       [  11.],
       [  15.],
       [   9.],
       [  16.],
       [  15.],
       [  11.],
       [   7.],
       [   9.],
       [   4.],
       [   6.],
       [   7.],
       [  10.],
       [   3.],
       [  10.],
       [   3.],
       [   8.],
       [   7.],
       [   9.],
       [   4.],
       [   8.],
       [   5.],
       [   8.],
       [   4.]], dtype=float32) is not JSON serializable
 * Running on http://0.0.0.0:5000/
 * Restarting with reloader
121.54.32.163 - - [08/Feb/2015 13:51:04] "POST /imageSearch HTTP/1.1" 200 -
121.54.32.174 - - [10/Feb/2015 07:58:44] "POST /imageSearch HTTP/1.1" 200 -
121.54.32.174 - - [10/Feb/2015 08:42:10] "POST /imageSearch HTTP/1.1" 200 -
85.25.43.94 - - [11/Feb/2015 14:51:59] "GET / HTTP/1.1" 404 -
85.25.43.94 - - [11/Feb/2015 14:52:00] "GET /robots.txt HTTP/1.1" 404 -
112.206.25.195 - - [14/Feb/2015 23:04:24] "POST /imageSearch HTTP/1.1" 200 -
 * Detected change in 'api.py', reloading
Unhandled exception in thread started by 
sys.excepthook is missing
lost sys.stderr
 * Restarting with reloader
112.206.25.195 - - [14/Feb/2015 23:10:56] "POST /imageSearch HTTP/1.1" 200 -
 * Detected change in 'api.py', reloading
 * Restarting with reloader
112.206.25.195 - - [14/Feb/2015 23:13:49] "POST /imageSearch HTTP/1.1" 200 -
 * Detected change in 'api.py', reloading
 * Restarting with reloader
 * Running on http://0.0.0.0:5000/
 * Restarting with reloader
121.54.32.171 - - [15/Feb/2015 10:32:32] "POST /imageSearch HTTP/1.1" 200 -
121.54.32.162 - - [15/Feb/2015 11:36:59] "POST /imageSearch HTTP/1.1" 200 -
 * Detected change in 'api.py', reloading
 * Restarting with reloader
 * Running on http://0.0.0.0:5000/
 * Restarting with reloader
121.54.32.163 - - [16/Feb/2015 00:55:28] "POST /imageSearch HTTP/1.1" 200 -
121.54.32.163 - - [16/Feb/2015 00:58:39] "POST /imageSearch HTTP/1.1" 200 -
 * Detected change in 'api.py', reloading
 * Restarting with reloader
 * Running on http://0.0.0.0:5000/
 * Restarting with reloader
180.191.77.3 - - [18/Feb/2015 22:42:22] "POST /imageSearch HTTP/1.1" 200 -
180.191.77.3 - - [18/Feb/2015 22:43:08] "POST /imageSearch HTTP/1.1" 200 -
49.150.153.119 - - [19/Feb/2015 00:40:44] "POST /imageSearch HTTP/1.1" 200 -
